backend/app/src/pong/entities/position.entity.ts

```diff
@@ -0,0 +1,39 @@
+export class Position {
```

Comment:
SHould be interfaces right ?

Reply:
@COTHSC 
"When to use classes: When you want to create objects that have actual function code in them, have a constructor for initialization, and/or you want to create instances of them with new. Also, for simple data objects, you can use classes for setting up default values. Another time you would want to use them is when you are doing type chec" https://stackoverflow.com/questions/40973074/difference-between-interfaces-and-classes-in-typescript

Reply:
Resolved!